src/tests/ie_test_utils/common_test_utils/test_common.cpp

```diff
@@ -23,6 +23,7 @@
 #endif
 
 namespace CommonTestUtils {
+const char* DEVICE_GPU = "GPU"; //declaration: test_constants.hpp
```

Comment:
Extra line! There is this constant in `test_constants.hpp` (const char DEVICE_GPU[] = "GPU";)

Reply:
Done.